### PR TITLE
VIH-6301 enable or disable camera in a hearing

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

VIH-6301

### Change description ###

* Allow participants (excluding judge) to enable or disable their camera in a hearing
* Replaced hearing control buttons with icons
judge-controls
non-judge-controls

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by virensareen — DISMISSED** [review]

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls.component.scss [hunk 1]
```diff
@@ -67,10 +77,71 @@
...
+}
+
+.toggle-video-mute-icon-btn,
+.pause-hearing-icon-btn {
+  grid-column: 3;
+  grid-row: 1;
+  opacity: 50%;
+}
+
+.toggle-video-mute-icon-btn:hover,
```
Comment (levibutler) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls.component.scss:128]: could these also be here?
.toggle-hand-raised-icon-btn:hover,
.end-hearing-icon-btn:hover

Comment (levibutler) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls.component.scss:128]: toggle-audio-mute-icon-btn:hover

Comment (levibutler) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls.component.scss:128]: or is something like this possible? - not tested
img[class^="toggle-"]:hover {
opacity: 100%;
}

Comment (shaed-parkar) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls.component.scss:128]: I'll try and see if I can create a base class which extends. I forgot to come back to this.

**Review by levibutler — APPROVED** [review]

**Review by levibutler — APPROVED** [review]

**Review by levibutler — DISMISSED** [review]

**Review by levibutler — APPROVED** [review]

**Pull request merged**